generator/README.md

```diff
@@ -24,6 +24,27 @@ The generator reads in from `generator.yml` and writes to `snmp.yml`.
 
 Additional command are available for debugging, use the `help` command to see them.
 
+## Docker Users
+
+If you would like to run the generator in docker to generate your `snmp.yml` config run the following commands.
+
+```
+# Build the image
+docker build -t snmp-generator .
+
+# The following command will:
```

Comment:
This detail is not relevant to the user. They just need to know where to put the mibs, and where the output will be.